index.js

```diff
@@ -54,7 +81,23 @@ function getHighestMetascore() {}
  *  getAverageIMDBRating(movies);
  *  //> 7.76
  */
-function getAverageIMDBRating() {}
+function getAverageIMDBRating(movies) {
+  //Return average of IMBD ratings which are strings that need to be converted into a number. Use Number() and add all ratings then divide by movies.length.
+  //If the inputted array is empty then return 0.
+
+  if(!movies || movies.length === 0) {
+    return 0;
+  };
+
+  let sumOfTotalImdbRatings = 0;
+  let averageTotalImdbRatings = 0;
+
+  for(let cinema of movies) {
+    sumOfTotalImdbRatings += Number(cinema.imdbRating);
+    averageTotalImdbRatings = sumOfTotalImdbRatings / movies.length;
```

Comment:
We don't quite need to keep reassigning `averageTotalImdbRatings` here. Instead, we could simply define it and assign it after we've accumulated our sum and right before we return it.